app/serializers/sidebar_section_serializer.rb

```diff
@@ -4,7 +4,7 @@ class SidebarSectionSerializer < ApplicationSerializer
   attributes :id, :title, :links, :slug, :public
 
   def links
-    object.sidebar_section_links.map(&:linkable)
+    object.sidebar_section_links.sort_by(&:position).map(&:linkable)
```

Comment:
Instead of sorting here, perhaps we can specify an order when fetching the links instead so that we can make use of the DB index.